code-studio/src/js/levels/levelGroup.js

```diff
@@ -68,7 +68,11 @@ window.initLevelGroup = function (
 
   var lastResponse = window.getResult().response;
 
-  window.levelGroup.answerChangedFn = function (levelId) {
+  window.levelGroup.answerChangedFn = function (levelId, commit) {
+    if (!commit) {
```

Comment:
Changing it to `saveThisAnswer` since this function uses it to decide whether or not to call `throttledSaveAnswers()`